### ISSUE TITLE
Fix plotting layout

### ISSUE THREAD

**Issue by dfm** [davidwhogg/TheThresher#38]
```
kernel --- decon*old  --- old  --- stretch-old
 PSF   ---    data    --- this --- stretch-this
 decon --- decon*this --- new  --- stretch-new
```

@drphilmarshall might disagree with the narrative.

Also, what about residuals?

**Comment by dfm** [comment]
residuals.

**Comment by dfm** [comment]
This should have all of the `scene`, `data`, etc. plots on the _same_ grayscale.

**Comment by dfm** [comment]
Fixed in cd92ac0c0de3420ade45369948ae9b59e0eb8483
